staker-rewards/lib/rewards-calculator.js

```diff
@@ -131,11 +195,13 @@ function OperatorRewards(
   ethScore,
   boost,
   rewardWeight,
-  totalRewards
+  totalRewards,
+  requirementsViolations
 ) {
   ;(this.operator = operator),
     (this.ethScore = ethScore),
     (this.boost = boost),
     (this.rewardWeight = rewardWeight),
-    (this.totalRewards = totalRewards)
+    (this.totalRewards = totalRewards),
+    (this.requirementsViolations = requirementsViolations)
```

Comment:
I am not sure if we need to list all violations in a separate column. It is pretty clear looking at `factoryAuthorizedAtStart`, `poolAuthorizedAtStart` and the rest what's wrong with the operator. I think the only problematic case is SLA which has more complex logic.

For that reason, what do you think about removing `requirementsViolations` column altogether and instead display `SLAViolated` column only for SLA? I am fine with having `requirementsViolations` field in a struct and using it in tests, I am just not sure if we need to print the value in the table given that this table already has enormous width.